docs/modules/ROOT/pages/tutorials/migration.adoc

```diff
@@ -0,0 +1,645 @@
+
+[source,yaml]
+--
+- apiVersion: route.openshift.io/v1
+  kind: Route
+  metadata:
+    name: ${DEPLOYMENT_NAME}
+    labels:
+      app: ${CI_ENVIRONMENT_SLUG}
+    annotations:
+      kubernetes.io/tls-acme: "${ROUTE_TLS_ACME}" # <1>
+  spec:
+    host: ${ROUTE_HOST}
+    port:
+      targetPort: http
+    tls:
+      insecureEdgeTerminationPolicy: Redirect
+      termination: edge
+    to:
+        kind: Service
+        name: ${DEPLOYMENT_NAME}
+        weight: 100
+    wildcardPolicy: None
+--
+<1> Annotation providing automatic provisioning of certificates on APPUiO Public.
+
+== Storage
+
+APPUiO Cloud has different requirements regarding storage. This section provides more information.
+
+=== Access the RWO volumes from different pods
+
+On APPUiO Public the default was RWX storage. At the moment, *APPUiO Cloud only features RWO storage*. Former deployments using RWX volumes can't run anymore without modification.
+
+NOTE: APPUiO Cloud will feature RWX volumes in the future, please refer to https://github.com/appuio/appuio-cloud-community/projects/1[this issue] for updates.
+
+When migrating your volumes to APPUiO Cloud, please follow these guidelines:
+
+. The rollout strategy must change from `RollingUpdate` to `Recreate`, because mounting the volume at different nodes simultaneously isn't possible, or at least not with common used filesystems, such as https://en.wikipedia.org/wiki/Ext4[ext4] and https://en.wikipedia.org/wiki/XFS[XFS]. https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy[This page] in the Kubernetes documentation provides more information about this.
+
+. Add an affinity rule to run on the same node as the worker pod. This allows to access the same RWO/block volume, because the filesystem is mounted on the node level, and allows a safe access to the same filesystem, because it's the same kernel.
++
+IMPORTANT: This could be used for jobs, but in the case of independent application parts this should be prevented.
++
+To create a affinity rule for all the pods at once, you can use the https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/[app.kubernetes.io/component label].
++
+[source,yaml]
+--
+affinity:
+  podAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+    - labelSelector:
+        matchExpressions:
+        - key: app.kubernetes.io/component
+           operator: In
+          values:
+          - backend
+      topologyKey: kubernetes.io/hostname
+--
+
+. Perform the migration itself. There are three major mechanisms for this:
+.. Migration with `rsync`
+.. Migration through jobs
+.. Using continuous sync
+
+The following sections provide information about each strategy.
+
+### Migration with `rsync`
+
+The first option to migrate your storage from APPUiO Public to APPUiO Cloud consists in using `rsync`. The manifests below create the required objects.
```

Comment:
This might need some more explanation: Where do I create these objects?

Reply:
I'll wait for the review by @megian to rewrite this, as this section comes directly from the text of his ticket #36

Reply:
This part has to be installed on the destination side. Overall a pod must be running with the RWO volume mounted, the òc rsync` can connect to.